shared/fs/row/container.js

```diff
@@ -26,8 +26,8 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     if (type === 'folder') {
       dispatch(navigateAppend([{props: {path}, selected: 'folder'}]))
     } else {
-      dispatch(FSGen.createDownload({path}))
-      console.log('Cannot view files yet. Requested file: ' + Types.pathToString(path))
+      //      dispatch(FsGen.createDownload({path}))
```

Comment:
@songgao: where should this go now?

Reply:
Thanks for tagging. I have it in my popup menu PR. It's OK to remove this now.